src/hotspot/share/gc/parallel/parallelArguments.cpp

```diff
@@ -70,7 +70,7 @@ void ParallelArguments::initialize() {
     if (FLAG_IS_CMDLINE(InitialSurvivorRatio)) {
       if (FLAG_IS_CMDLINE(MinSurvivorRatio)) {
         jio_fprintf(defaultStream::error_stream(),
-          "Inconsistent MinSurvivorRatio vs InitialSurvivorRatio: %d vs %d\n", MinSurvivorRatio, InitialSurvivorRatio);
+          "Inconsistent MinSurvivorRatio vs InitialSurvivorRatio: %zu vs %zu\n", MinSurvivorRatio, InitialSurvivorRatio);
```

Comment:
Should probably use `UINTX_FORMAT` instead.

Reply:
Nope: https://bugs.openjdk.org/browse/JDK-8346990 Remove INTX_FORMAT and UINTX_FORMAT macros